ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt

```diff
@@ -147,139 +141,123 @@ hideall "--sync--"
-1011.5 "Polarizing Paths" Ability { id: "9D2A", source: "Pandora" }
-1016.1 "Polarizing Paths" Ability { id: "9D2A", source: "Pandora" }
-1020.7 "Polarizing Paths" Ability { id: "9D2A", source: "Pandora" }
-1040.9 "Pandora's Box" Ability { id: "9D86", source: "Pandora" }
-1053.0 "Fulgent Blade" Ability { id: "9D72", source: "Pandora" }
-1064.1 "The Path of Darkness/The Path of Light" Ability { id: "9CB6", source: "Pandora" } duration 19.7
-1079.3 "Akh Morn" Ability { id: "9D76", source: "Pandora" }
-1079.3 "Akh Morn" Ability { id: "9D77", source: "Pandora" }
-1079.3 "Akh Morn" Ability { id: "9D78", source: "Pandora" }
-1091.5 "Paradise Regained" Ability { id: "9D7F", source: "Pandora" }
-1101.5 "Wings Dark and Light" Ability { id: "9D29", source: "Pandora" }
-1101.8 "Explosion" Ability { id: "9D80", source: "Pandora" }
-1101.8 "Wings Dark and Light" Ability { id: "9D7B", source: "Pandora" }
-1102.7 "Wings Dark and Light" Ability { id: "9BC8", source: "Pandora" }
-1105.4 "Explosion" Ability { id: "9D80", source: "Pandora" }
-1105.6 "Wings Dark and Light" Ability { id: "9D7A", source: "Pandora" }
-1106.5 "Wings Dark and Light" Ability { id: "9BC7", source: "Pandora" }
-1108.9 "Explosion" Ability { id: "9D80", source: "Pandora" }
-1117.8 "Polarizing Strikes" Ability { id: "9D7C", source: "Pandora" }
-1118.4 "Cruel Path of Darkness/Cruel Path of Light" Ability { id: "9D7D", source: "Pandora" } duration 16.2
-1122.5 "Polarizing Paths" Ability { id: "9D2A", source: "Pandora" }
-1127.2 "Polarizing Paths" Ability { id: "9D2A", source: "Pandora" }
-1131.9 "Polarizing Paths" Ability { id: "9D2A", source: "Pandora" }
-1143.1 "Fulgent Blade" Ability { id: "9D72", source: "Pandora" }
-1154.2 "The Path of Darkness/The Path of Light" Ability { id: "9CB6", source: "Pandora" } duration 17.7
-1169.4 "Akh Morn" Ability { id: "9D76", source: "Pandora" }
-1169.4 "Akh Morn" Ability { id: "9D77", source: "Pandora" }
-1169.4 "Akh Morn" Ability { id: "9D78", source: "Pandora" }
-1189.6 "Paradise Lost" Ability { id: "9D87", source: "Pandora" }
+# Depending on where the Path of Darkness/Light starts, the duration can range from 18.3s to 22.5s
+# The tail end of this is largely irrelevant since the abilities are used at the far perimeter of the arena.
+# But to simplify, use a duration of 20.4 (the midpoint).
+1052.0 "The Path of Darkness + The Path of Light" Ability { id: "9CB6", source: "Pandora" } duration 20.4
+1067.8 "Akh Morn" Ability { id: "9D76", source: "Pandora" }
+1076.0 "Paradise Regained" Ability { id: "9D7F", source: "Pandora" }
+1086.0 "Wings Dark and Light" Ability { id: ["9D29", "9D79"], source: "Pandora" }
+1086.4 "Explosion" Ability { id: "9D80", source: "Pandora" }
+1089.9 "Wings Dark and Light + Explosion" Ability { id: "9D80", source: "Pandora" }
+1093.4 "Explosion" Ability { id: "9D80", source: "Pandora" }
+1107.4 "Polarizing Strikes" Ability { id: "9D7C", source: "Pandora" }
```

Comment:
The clear log I tested against showed this sync might be a bit off:

```
      -0.012 |  236 | 1093.4 "Explosion" Ability { id: "9D80", source: "Pandora" }
      -0.230 |  237 | 1107.4 "Polarizing Strikes" Ability { id: "9D7C", source: "Pandora" }
      +0.021 |  238 | 1108.0 "Cruel Path of Darkness + Cruel Path of Light" Ability { id: "9D7D", source: "Pandora" } duration 2.5
```

Reply:
Here's mine from yesterday:
```
      -0.018 |  237 | 1093.4 "Explosion" Ability { id: "9D80", source: "Pandora" }
      -0.071 |  238 | 1107.4 "Polarizing Strikes" Ability { id: "9D7C", source: "Pandora" }
      +0.020 |  239 | 1108.0 "Cruel Path of Darkness + Cruel Path of Light" Ability { id: "9D7D", source: "Pandora" } duration 2.5
```

And I think I have another that drifts in the opposite direction.  There are a number of mechs (a bunch in P3 for some reason) where I saw drift across logs, even in the same instance on different pulls that had variability of 0.4s or more, so some of this is an attempt to average out the differences.